yarn.lock

```diff
@@ -4,12 +4,12 @@
 
 "@ava/babel-plugin-throws-helper@^3.0.0":
   version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@ava/babel-plugin-throws-helper/-/babel-plugin-throws-helper-3.0.0.tgz#2c933ec22da0c4ce1fc5369f2b95452c70420586"
+  resolved "https://registry.npmjs.org/@ava/babel-plugin-throws-helper/-/babel-plugin-throws-helper-3.0.0.tgz"
```

Comment:
Any reason why there are so many changes to this file? Perhaps we used `npm install` instead of `yarn install`? Unless these yarn changes are required for the change, I would remove them.